framework/test/unit/modules/dynamic_rewards/endpoint.spec.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { utils, address as cryptoAddress } from '@liskhq/lisk-cryptography';
+import { DynamicRewardModule } from '../../../../src/modules/dynamic_rewards';
+import { DynamicRewardEndpoint } from '../../../../src/modules/dynamic_rewards/endpoint';
+import { PrefixedStateReadWriter } from '../../../../src/state_machine/prefixed_state_read_writer';
+import {
+	InMemoryPrefixedStateDB,
+	createFakeBlockHeader,
+	createTransientModuleEndpointContext,
+} from '../../../../src/testing';
+
+describe('DynamicRewardModuleEndpoint', () => {
+	const genesisConfig: any = { blockTime: 7 };
+	const moduleConfig: any = {
+		distance: 3000000,
+		offset: 2160,
+		brackets: [
+			'500000000', // Initial Reward
+			'400000000', // Milestone 1
+			'300000000', // Milestone 2
+			'200000000', // Milestone 3
+			'100000000', // Milestone 4
+		],
+		tokenID: '0000000000000000',
+		factorMinimumRewardActiveValidators: 1000,
+	};
+
+	const address = utils.getRandomBytes(20);
+	const address1 = utils.getRandomBytes(20);
```

Comment:
I think `address1` is not used in the tests, so probably can be removed

Reply:
this is used in the fixture 👍